electrumsv/storage.py

```diff
@@ -123,6 +123,12 @@ def get_categorised_files(wallet_path: str, exclude_suffix: str='') -> list[Wall
         s for s in os.listdir(wallet_path)
         if not (exclude_suffix and s.endswith(exclude_suffix))
     }
+    # Exclude this directory - these wallets have the password set as 'test' which would
```

Comment:
Please get rid of the workaround and understand why it is not working and make it work. Get rid of the new wallet subdirectory added to support this workaround as well.

Reply:
Either I need to fix the bug that renders a database file unable to be opened after changing password or I need to redo the creation of the regtest wallet from the beginning with the password as `123456` instead of `test`

I am going to investigate now exactly what the issue is with changing wallet password and how hard it is to fix.

Reply:
Okay this PR: https://github.com/electrumsv/electrumsv/pull/1070 has now allowed me to successfully update the wallet password of the test wallet databases and move them back to the top-level `tests/data/wallets/directory`